EnglishFilter/sections/antiadblock.txt

```diff
@@ -399,6 +399,8 @@ ask4movie.co,airlive.net,anonyviet.com,arch2o.com,asianhobbyist.com,bpmodyssee.c
 ||ask4movie.co/wp-content/cache/minify/7f26c.js
 ! https://github.com/AdguardTeam/AdguardFilters/issues/36036
 nekopoi.care#%#AG_setConstant('popjs', 'noopFunc'); 
+! https://github.com/AdguardTeam/AdguardFilters/issues/35714
+sinagritbabasworkshop.com##body > #mbd1
```

Comment:
For reference (since one rule is better than those many, please do not add them):
```
sinagritbabasworkshop.com#@##box-ads-small-1
sinagritbabasworkshop.com#@##adsense
sinagritbabasworkshop.com#@##themis-ads
sinagritbabasworkshop.com#@##sponsored_v12
sinagritbabasworkshop.com#@##pcworldAdTop
sinagritbabasworkshop.com#@##medRecAd
sinagritbabasworkshop.com#@##fl_hdrAd
sinagritbabasworkshop.com#@##firstad
```